lenskit-eval/build.gradle

```diff
@@ -39,6 +39,7 @@ apply from: "$rootDir/gradle/test-utils.gradle"
 
 dependencies {
     compile project(':lenskit-core')
+    compile project(':lenskit-knn')
```

Comment:
Got it. I think I see why — baselines will still predict for unknown users. This error likely arises when only when no users could be predicted.